app/settings.py

```diff
@@ -32,7 +32,7 @@ def get_key(_key_name):
 EQ_MAX_HTTP_POST_CONTENT_LENGTH = os.getenv('EQ_MAX_HTTP_POST_CONTENT_LENGTH', 65536)
 
 # max number of repeats from a rule
-EQ_MAX_NUM_REPEATS = os.getenv('EQ_MAX_NUM_REPEATS', 50)
+EQ_MAX_NUM_REPEATS = os.getenv('EQ_MAX_NUM_REPEATS', 25)
```

Comment:
While we didn't plan to change this with this PR, I'm happy for this to be 25 by default as a sensible number for now.